src/Tizen.NUI.Scene3D/src/public/ModelMotion/MotionValue.cs

```diff
@@ -119,7 +119,7 @@ internal MotionValue(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, c
         /// Get or set the value as PropertyValue type.
         /// If ValueType is ValueType.KeyFrames, it will return last value of stored KeyFrames.
```

Comment:
it should return PropertyValue instance, not KeyFrames

Reply:
I change the sentence more clearly.